openlibrary/core/lending.py

```diff
@@ -337,16 +337,14 @@ def get_availability(key, ids):
         return {}
 
     def update_availability_schema_to_v2(v1_resp, ocaid):
+        """This functionattempts to take the output of e.g. Bulk Availability
+        API and add/infer attributes which are missing (but are
+        present on Ground Truth API)
+        """
+        # TODO: Make less brittle; maybe add simplelists/copy counts to Bulk Availability
         v1_resp['identifier'] = ocaid
         v1_resp['is_restricted'] = v1_resp['status'] != 'open'
-        v1_resp['is_printdisabled'] = v1_resp.get('is_printdisabled', False)
-        v1_resp['is_lendable'] = v1_resp['status'] == 'borrow_available'
```

Comment:
This line is flat out wrong -- `borrow_unavailable` can also imply `is_lendable`. And besides, the value is already coming from the Bulk Availability API now (w/ latest schema) so this adapter is no longer necessary. Same for most the removed fields.